packages/lodestar/src/network/util.ts

```diff
@@ -110,3 +112,19 @@ export function prettyPrintPeerId(peerId: PeerId): string {
 export function getAgentVersionFromPeerStore(peerId: PeerId, metadataBook: MetadataBook): string {
   return new TextDecoder().decode(metadataBook.getValue(peerId, "AgentVersion")) || "N/A";
 }
+
```

Comment:
@dapplion I moved the `getCurrentAndNextFork` over here so that it can be used for `AttestationServices`, made some small changes and unit tests for it

Reply:
Looks good! On demand from cayman config.forks are sorted in ascending order. We can use that fact to prevent having to sort the forks everytime. So passing `config` to forks instead of an array of ForkInfo